tests/Database/DatabaseEloquentBelongsToManySyncReturnValueTypeTest.php

```diff
@@ -90,7 +90,7 @@ public function testSyncReturnValueType()
         });
 
         $user->articles->each(function (BelongsToManySyncTestTestArticle $article) {
-            $this->assertSame('0', $article->pivot->visible);
+            $this->assertEquals('0', $article->pivot->visible);
```

Comment:
The reason here is that sqlite now returns integers instead of strings for integer valued columns, in PHP 8.1.